examples/dbscan/ArborX_DBSCAN.hpp

```diff
@@ -236,60 +236,70 @@ void dbscan(ExecutionSpace exec_space, Primitives const &primitives,
   start = clock::now();
   Kokkos::Profiling::pushRegion("ArborX::DBSCAN::sort_and_filter_clusters");
 
+  Kokkos::View<int *, MemorySpace> cluster_sizes(
+      "ArborX::DBSCAN::cluster_sizes", n);
+  Kokkos::parallel_for("ArborX::DBSCAN::compute_cluster_sizes",
+                       Kokkos::RangePolicy<ExecutionSpace>(exec_space, 0, n),
+                       KOKKOS_LAMBDA(int const i) {
+                         Kokkos::atomic_fetch_add(&cluster_sizes(clusters(i)),
+                                                  1);
+                       });
```

Comment:
I don't think using `ScatterView` would helpful here. My understanding is that it is mostly helpful when used with small sized views. Here, on the other hand, we computer cluster sizes but store then in sparse array, which would be duplicated.

Reply:
I'll switch to using `ScatterView`, but later, as at the moment SYCL backend does not support them, and I want us to use this driver for testing.